atlassian-labs/data-center-terraform
# TFKUBE-360: Fixes for issues identified during Confluence blitz

Various fixes for issues that were identified during the Confluence blitz.

- Fix synchrony URL for deployments that do not use a domain.
- e2e test to verify synchrony is up
- Update cpu / memory request sizing
- Add the RDS flag [apply_immediately](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#apply_immediately)  - Specifies whether any database modifications are applied immediately, or during the next maintenance window. Default is false. See [Amazon RDS Documentation for more information.](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html)
